cmd/neofs-cli/modules/object/lock.go

```diff
@@ -21,22 +21,22 @@ import (
 
 // object lock command.
 var objectLockCmd = &cobra.Command{
-	Use:   "lock CONTAINER OBJECT...",
+	Use:   "lock",
 	Short: "Lock object in container",
 	Long:  "Lock object in container",
-	Args:  cobra.MinimumNArgs(2),
-	Run: func(cmd *cobra.Command, args []string) {
-		var cnr cid.ID
+	Run: func(cmd *cobra.Command, _ []string) {
+		cidRaw, _ := cmd.Flags().GetString("cid")
 
-		err := cnr.DecodeString(args[0])
+		var cnr cid.ID
+		err := cnr.DecodeString(cidRaw)
 		common.ExitOnErr(cmd, "Incorrect container arg: %v", err)
 
-		argsList := args[1:]
+		oidsRaw, _ := cmd.Flags().GetStringSlice("oid")
```

Comment:
We need a named const for cid and oid, they are used 3 times.

Reply:
https://github.com/nspcc-dev/neofs-node/pull/1924#discussion_r1001618644

All of our pkgs are facing that problem, suggest fixing that in a separate PR, will rebase that PR if the PR from the link is merges before